bin/zField

```diff
@@ -150,8 +150,37 @@ if __name__ == '__main__':
             dMapDict = clusters.makeDensityMap(RADeg, decDeg, galaxyCatalog, z, dz, rMaxMpc = 1.5)
             astImages.saveFITS(args.outDir+os.path.sep+"densityMap_z%s_%.3f_%.3f.fits" % (z, RADeg, decDeg),
                                dMapDict['map'], dMapDict['wcs'])
-    
+            
+    #estimate errors for density map, CS and A
```

Comment:
Please follow the style of the rest of the code - e.g, here  # Estimate errors ... (note space and capital).